202009.0/ffa6653a-7d0e-41cc-a24e-074752b2ce1e.md

```diff
@@ -58,31 +50,8 @@ Depending on your requirements, you can select any combination of the following
 - `--assets` - build assets
 - `--data` - fetch new demo data
 
-
-## Setting up a production-like environment
-
-To set up a production-like environment:
-
-1. Bootstrap docker setup, build and start the instance:
-
-```bash
-docker/sdk boot deploy.*.yml
-docker/sdk up
-```
-
-2. Switch to your project branch, re-build the application with assets and demo data from the new branch:
-
-```bash
-git checkout {your_branch_name}
-docker/sdk boot
-docker/sdk up --build --assets --data
-```
-
-Depending on your requirements, you can select any combination of the following `up` command attributes. To fetch all the changes from the branch you switch to, we recommend running the command with all of them:
-- `--build` - update composer, generate transfer objects, etc.
-- `--assets` - build assets
-- `--data` - get new demo data
-
+Here you go! Now every Spryker Application of your project is running in a separate Docker container and
```

Comment:
@sprymiker could you add, how can I confirm a successful installation,
what links to use? or maybe a healthcheck run?